plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -988,13 +992,14 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
       duration = animationTime
     }
 
-    zoomAnimator.addListener(object : AnimatorListenerAdapter() {
+    anchorAnimator.addListener(object : AnimatorListenerAdapter() {
 
       override fun onAnimationStart(animation: Animator) {}
 
       override fun onAnimationCancel(animation: Animator) {}
 
       override fun onAnimationEnd(animation: Animator) {
+        cameraAnimationsPlugin.resetAnchor()
```

Comment:
When the gesture finishes it shouldn't reset the anchor to map center but rather to the value that was there previously, right?

This is what is proposed:
1. User transitions anchor from X -> Y
2. Gesture starts and transitions the anchor from Y -> Z
3. Gesture ends and transitions the anchor from Z -> map_center

This looks incorrect to me, I think it should rather be:
1. User transitions anchor from X -> Y
2. Gesture starts and transitions the anchor from Y -> Z
3. Gesture ends and transitions the anchor from Z -> Y

Reply:
Good catch, it means we should actually expose `cameraAnimationsPlugin.getAnchor()` as I did initially but then decided reset is enough 😄 
Will do!